src/test/resources/ProgramAB/bots/pikachu/aiml/pikachu.aiml

```diff
@@ -1,10 +1,8 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<aiml>
-	<category>
-		<pattern>私は*です</pattern>
-		<!--  あなたを*に会うのはいいことだ   ?? -->
-		<!--  あなたを*に会えてうれしい -->
-		<!-- あなたに会えてよかったXXXXX -->
-		<template>あなたに会えてよかった<star /></template>
-	</category>
-</aiml>
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?>
+<aiml>
+<!--  -->
+<!--  -->
+<category><pattern>私 は * です</pattern>
```

Comment:
your auto-formatting actually broke this!  there should not be spaces in the pattern tag here.  Please revert this change.

Reply:
reverted

Reply:
Thanks, Japanese text does not have spaces between words.  I have to noodle on how a unit test should work to verify that.  Also, if anyone speaks Japanese, can they verify my grammar here.